container.te

```diff
@@ -1033,3 +1033,33 @@ dontaudit container_domain device_node:chr_file setattr;
 dontaudit container_domain sysctl_type:file write;
 
 allow container_t proc_t:filesystem remount;
+
+
+########################################
+#
+# virt_launcher_t local policy
+#
+# SELinux security policy for kubevirt project
+# Temporary solution for rhbz#1795975
+
+type virt_launcher_t, container_domain;
+domain_type(virt_launcher_t)
+
+allow virt_launcher_t self:tun_socket { attach_queue relabelfrom relabelto create_socket_perms };
+
+manage_dirs_pattern(virt_launcher_t, container_var_run_t, container_var_run_t)
```

Comment:
This looks wrong, is this a labeling issue?  IE are we mounting content in /run created by the container runtime, without a :Z?

Reply:
`/var/run/kubevirt-infra/healthy` is being written to by the virt-launcher process. Since that's technically a new file created after the container is running, would that be why this is needed?

Reply:
@stu-gott are you bind mounting this file to the container space?

Reply:
Yes I believe so. virt-handler reports health on behalf of virt-launcher (which is isolated from cluster resources), thus the bind mount.

Reply:
@rhatdan  do we have something like :Z for bindmounts for kubernetes?

Reply:
You can add this to the destination, I believe, and then CRI-O will relabel the content.

Reply:
I think it would be better if we left KubeVirt as-is for the moment. Otherwise we get into this weird situation where we need a compatibility matrix to know what versions of kubevirt will run on what versions of RHCOS.